File: cpp/1048.cpp
#include <iostream>
#include <stdio.h>

using namespace std;

int main()
{
    float salary, multiplier;
    cin >> salary;

    if (salary <= 400) {
        multiplier = 15;
    } else if (salary <= 800) {
        multiplier = 12;
    } else if (salary <= 1200) {
        multiplier = 10;
    } else if (salary <= 2000) {
        multiplier = 7;
    } else {
        multiplier = 4;
    }

    printf("Novo salario: %.2f\n", salary * (1 + multiplier / 100));
    printf("Reajuste ganho: %.2f\n", salary * (multiplier / 100));
    printf("Em percentual: %d %\n", (int)(multiplier));
    return 0;
}

File: py/1048.py
salary = float(input())

if salary <=  400:
    print(f'Novo salario: {(salary * 1.15):.2f}')
    print(f'Reajuste ganho: {(salary * 0.15):.2f}')
    print(f'Em percentual: 15 %')
    
elif salary <= 800:
    print(f'Novo salario: {(salary * 1.12):.2f}')
    print(f'Reajuste ganho: {(salary * 0.12):.2f}')
    print(f'Em percentual: 12 %')
    
elif salary <= 1200:
    print(f'Novo salario: {(salary * 1.10):.2f}')
    print(f'Reajuste ganho: {(salary * 0.10):.2f}')
    print(f'Em percentual: 10 %')

elif salary <= 2000:
    print(f'Novo salario: {(salary * 1.07):.2f}')
    print(f'Reajuste ganho: {(salary * 0.07):.2f}')
    print(f'Em percentual: 7 %')
    
else:
    print(f'Novo salario: {(salary * 1.04):.2f}')
    print(f'Reajuste ganho: {(salary * 0.04):.2f}')
    print(f'Em percentual: 4 %')